Makefile

```diff
@@ -63,8 +63,8 @@ uninstall:
 	$(call UNINSTALL_LAB_EXTENSION,@elyra/pipeline-editor-extension)
 	$(call UNINSTALL_LAB_EXTENSION,@elyra/python-editor-extension)
 	$(call UNINSTALL_LAB_EXTENSION,@elyra/r-editor-extension)
-	- jupyter labextension uninstall @elyra/pipeline-services
-	- jupyter labextension uninstall @elyra/pipeline-editor
+	- jupyter labextension unlink @elyra/pipeline-services
+	- jupyter labextension unlink @elyra/pipeline-editor
```

Comment:
will this still work with you don't have a linked extension?

Reply:
that's what the `-` is for, it tells make to ignore failure for that line and continue on.